README.md

```diff
@@ -15,20 +15,20 @@ A [Terraform] base module for creating an
 ***This module supports Terraform v1.x, v0.15, v0.14, v0.13 as well as v0.12.20 and above
 and is compatible with the Terraform AWS provider v3 as well as v2.45 and above.***
 
+
 - [Module Features](#module-features)
 - [Getting Started](#getting-started)
+  - [Access ECR with IAM principals](#access-ecr-with-iam-principals)
 - [Module Argument Reference](#module-argument-reference)
   - [Top-level Arguments](#top-level-arguments)
+    - [Module Configuration](#module-configuration)
     - [Main Resource Configuration](#main-resource-configuration)
     - [Extended Resource configuration](#extended-resource-configuration)
     - [Additional configuration](#additional-configuration)
-    - [`policy_statements` Object Arguments](#policy_statements-object-arguments)
-    - [`principal` Object Arguments](#principal-object-arguments)
-    - [`lifecycle_rules` Object Arguments](#lifecycle_rules-object-arguments)
-    - [`selection` Object Arguments](#selection-object-arguments)
-    - [`action` Object Arguments](#action-object-arguments)
```

Comment:
why is this getting removed? is this expected?

Reply:
@soerenmartius Yes it is. The arguments have been moved from their own extra sections like in the above table of contents, to the variables themselves. It's all still there, just no longer as an extra section.